### ISSUE TITLE
Issue when using orchestra/platform installer 

### ISSUE THREAD

**Issue by laksamanakeris** [orchestral/platform#63]
When I install using orchestra/platform installer

```
composer create-project orchestra/platform 3.3.x --prefer-dist --stability=dev
```
 
I get

```
Writing lock file
Generating autoload files
> Orchestra\Foundation\ComposerScripts::postUpdate
> php artisan orchestra:assemble


  [ErrorException]
  Declaration of App\Providers\EventServiceProvider::boot() should be compatible with Illuminate\Foundation\Support\Providers\EventServiceProvider::boot()


Script php artisan orchestra:assemble handling the post-update-cmd event returned with an error


  [RuntimeException]
  Error Output:

```

**Comment by crynobone** [comment]
Should already be fixed 4b91b104

**Comment by laksamanakeris** [comment]
Awesome. Thank you!
